tests/test_monitor.py

```diff
@@ -320,3 +326,337 @@ def test_get_machine_info(**kwargs):
         actual_machine_info_output = get_machine_info(compute)
 
     assert actual_machine_info_output == expected_machine_info_output
+
+
+def get_services_pricelist(filepath: str) -> List[dict]:
+    with open(filepath) as f:
+        return json.load(f)
+
+
+def get_units_and_nanos(sku_id: str, pricelist: List[dict]) -> tuple[int, int]:
+    sku = [sku for sku in pricelist if sku_id in sku["skuId"]][0]
```

Comment:
more pythonic (and performant)
```suggestion
    sku = next(sku for sku in pricelist if sku_id in sku["skuId"])
```